udf-compiler/src/main/scala/com/nvidia/spark/udf/Plugin.scala

```diff
@@ -26,7 +26,7 @@ import org.apache.spark.sql.catalyst.rules.Rule
 class Plugin extends (SparkSessionExtensions => Unit) with Logging {
   override def apply(extensions: SparkSessionExtensions): Unit = {
     logWarning("Installing rapids UDF compiler extensions to Spark. The compiler is disabled" +
```

Comment:
IMO if the UDF compiler is disabled, we shouldn't print any message at all about it on startup.  A startup message about the UDF compiler should appear iff the UDF compiler is enabled.

Reply:
+1 and I would leave the config in the log line also, but turn this around to be:

> To disable it, set `spark.rapids.sql.udfCompiler.enabled` to false

Reply:
I tried to get the `RapidsConf.isUdfCompilerEnabled`, but failed because I could not get an active Spark Session
```scala
  override def apply(extensions: SparkSessionExtensions): Unit = {
    SparkSession.getActiveSession match {
      case None => // no active session
      case Some(session) => // log warning based on `session.sparkContext.getConf.isUdfCompilerEnabled`
    }
    extensions.injectResolutionRule(logicalPlanRules)
  }
```

Reply:
There's no Spark session this early in startup.  We'll either need to move the logging message to the code that is injecting the plugin (see `RapidsPluginUtils.fixupConfigs`) which does have access to the configs, or the configs can be fetched from `SparkEnv.get.conf`.  I slightly prefer the former since the configs are right there.

Reply:
Done!